src/defines.h

```diff
@@ -181,4 +181,12 @@ IMU_DESC_ENTRY(IMU_BMP160, PRIMARY_IMU_ADDRESS_ONE, IMU_ROTATION, PIN_IMU_SCL, P
   #define PIN_BATTERY_LEVEL 3
   #define LED_PIN LED_OFF  // RGB LED Protocol would need to be implementetet did not brother for the test, because the board ideal for tracker ifself
 //  #define LED_INVERTED false
+#elif BOARD == BOARD_WROOM02
```

Comment:
```suggestion
#elif BOARD == BOARD_WEMOSWROOM02
```

I dont want to mix it up with the ESP WROOM02 blank module.